core/src/components/range/range.md.scss

```diff
@@ -16,19 +16,24 @@
   --height: #{$range-md-slider-height};
   --pin-background: #{ion-color(primary, base)};
   --pin-color: #{ion-color(primary, contrast)};
-
-  // TODO FW-2997 Apply this to the start/end slots, and the native wrapper
-  font-size: $range-md-pin-font-size;
 }
 
 // TODO FW-2997 Remove this
-::slotted([slot="label"]) {
+:host(.legacy-range) ::slotted([slot="label"]) {
   font-size: initial;
 }
 
+:host(:not(.legacy-range)) ::slotted(:not(ion-icon)[slot="start"]),
+:host(:not(.legacy-range)) ::slotted(:not(ion-icon)[slot="end"]),
+:host(:not(.legacy-range)) .native-wrapper {
+  font-size: $range-md-pin-font-size;
+}
+
```

Comment:
This is pretty gnarly, but I basically wanted to apply the smaller font size to the pin and text in the start/end slots (excluding icons). Any ideas for how I can do this better?

Reply:
You could target the icon after but then we would be always overriding icon size:

```
:host(:not(.legacy-range)) ::slotted(*),
:host(:not(.legacy-range)) .native-wrapper {
  font-size: $range-md-pin-font-size;
}

:host(:not(.legacy-range)) ::slotted(ion-icon[slot]) {
  font-size: 24px;
}
```

On that note - are the icons supposed to respond to dynamic type?

Reply:
I opted not to since native doesn't seem to do it: 
![IMG_2579](https://github.com/ionic-team/ionic-framework/assets/2721089/1034c072-63f0-41b7-88ca-1700ecb893ea)

Reply:
Oh good call. Is there a native example with text?

Reply:
![IMG_2581](https://github.com/ionic-team/ionic-framework/assets/2721089/c38d8245-c908-499c-833d-18505ecf2b95)

This is the closest I could find

Reply:
Thanks for that example! So I did some comparisons and it looks like it should be using the following:

```
font-size: max(14px, 17px); 
```

which is roughly:

```
font-size: dynamic-font-min(0.82, 17px);
```

However, since it is currently using `16px` we probably don't want to update it, so I think it's fine as is.

Reply:
As for the original question, I think it's fine unless you want to condense it like:

```
:host(:not(.legacy-range)) ::slotted(:not(ion-icon)[slot]),
:host(:not(.legacy-range)) .native-wrapper {
  font-size: $range-md-pin-font-size;
}
```